parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -450,6 +450,19 @@ public static final ParquetMetadata readFooter(Configuration configuration, File
    * @throws IOException if an error occurs while reading the file
    */
   public static final ParquetMetadata readFooter(long fileLen, String filePath, SeekableInputStream f, MetadataFilter filter) throws IOException {
+    return readFooter(new ParquetMetadataConverter(), fileLen, filePath, f, filter);
```

Comment:
We should not pass hadoop configurations around. see my comments on #368.

Regarding the config I was just suggesting using it in this particular method and that particular setting. Since it has not been released yet.
Once this method is released changing its signature will require deprecating it.
But adding fields to the ParquetReadConfig object will be backwards compatible.

the parameterless ParquetMetadataConverter() should be deprecated